python/az/aro/azext_aro/_params.py

```diff
@@ -90,3 +91,12 @@ def load_arguments(self, _):
         c.argument('worker_subnet',
                    help='Name or ID of worker vnet subnet.  If name is supplied, `--vnet` must be supplied.',
                    validator=validate_subnet('worker_subnet'))
+    with self.argument_context('aro update') as c:
+        c.argument('client_secret',
+                   help='Client secret of cluster service principal.',
+                   validator=validate_client_secret(isCreate=None))
```

Comment:
nit - for consistency.  
```suggestion
                   validator=validate_client_secret(isCreate=False))
```